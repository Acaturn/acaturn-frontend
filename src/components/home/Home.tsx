// tslint:disable-next-line
import React from 'react';
import './Home.scss';

const Home = () => (
  <div className="home">
    <h1>Welcome to Acaturn</h1>
  </div>
);

export default Home;
