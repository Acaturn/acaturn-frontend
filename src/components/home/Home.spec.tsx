import React from 'react';
import { shallow } from 'enzyme';
import Home from './Home';

describe('Home', () => {
  it('should render home page', () => {
    const home = shallow(<Home />);
    expect(home).toMatchSnapshot();
  });
});
