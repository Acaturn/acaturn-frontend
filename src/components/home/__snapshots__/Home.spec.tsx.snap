// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render home page 1`] = `ShallowWrapper {}`;
